packages/beacon-node/test/spec/general/bls.ts

```diff
@@ -169,5 +189,9 @@ function sign(input: {privkey: string; message: string}): string | null {
  */
 function verify(input: {pubkey: string; message: string; signature: string}): boolean {
   const {pubkey, message, signature} = input;
-  return bls.verify(fromHexString(pubkey), fromHexString(message), fromHexString(signature));
+  try {
+    return _verify(fromHexString(message), fromHexString(pubkey), fromHexString(signature));
+  } catch {
+    return false;
+  }
```

Comment:
@wemeetagain I added this try/catch for the spec tests but am updating the base lib to not throw for this so it better meets the spec.